bin/setup

```diff
@@ -35,6 +35,10 @@ chdir APP_ROOT do
     cp 'config/secrets.yml.template', 'config/secrets.yml'
   end
 
+  unless File.exist?('config/smtp.yml')
```

Comment:
Should we be adding this config file during setup? If the config exists mail will attempt to send even if they haven't populated any details.

I think it's safer and fine to not automatically apply the smtp config on setup. We have the sample file, and the documentation regarding it's application.

Reply:
Actually- The user still has to setup cronjobs for emails to be triggered to send. So even if the SMTP config is blank in CE it won't change anything until the use specifically goes to setup emails, in which case they'll add settings. So I think this will be fine.